kube-app-testing.sh

```diff
@@ -515,7 +706,15 @@ parse_args () {
         CLUSTER_TYPE=$2
         shift 2
         ;;
-      *) 
+      -a|--auth-token)
+        GSAPI_AUTH_TOKEN=$2
+        shift 2
+        ;;
+      -r|--release-version)
+        GS_RELEASE=$2
+        shift 2
```

Comment:
I've not done argument parsing in bash this way before - please shout if this isn't going to work (I think it's ok though).

Reply:
`shift` just shifts left the  args array by the number of entries that is the parameter. In other words, `shift 2` deletes 2 first entries in the args array. So everything seems fine here :)

Reply:
sure, after some experimenting and googling i figured this out :+1: